manage-me-frontend/src/components/projects/columns.ts

```
@@ -0,0 +1,43 @@
+      return h(
+        'div',
+        { class: 'text-right font-medium' },
+        row.getValue('title')
+      );
+    },
+  },
+  {
+    accessorKey: 'description',
+    header: () => h('div', { class: 'text-right' }, 'Description'),
+    cell: ({ row }) => {
+      const rawDescription = row.getValue('description') as String;
```

Biome (1.9.4)

[error] Don't use 'String' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'string' instead


(lint/complexity/noBannedTypes)